### DISCUSSION TITLE
Suggesting additions to OSM-ISSprom CRT

### DISCUSSION THREAD

**Discussion by Choreuomai** [OpenOrienteering/mapper#2022] [Questions & Answers] [answered]
Hi,
I'm new to Github... What would be the right place/way to suggest additions for the OSM-ISSprom CRT table? I suppose opening an issue for just one or two ideas for lines to be added would not be right?

**Comment by dg0yt** [comment, marked as answer]
You can suggest it here if you want. Contributions in this field are very welcome.

(Issue or discussion, someone has to put in a pull request in the end. You may even start by "editing" the file here. Github will first create a "fork" of our repository in your area, and then save the change in a new branch there. In another step you would create the pull request. Later edits to your branch would update the pull request as long as it is not merged.)

**Reply by Choreuomai** [reply]
Well, just thought that I would add "401 landuse=residential AND residential=apartments" because of mostly open spaces in between, and yellow is below anyway. I have got more ideas that I am already using but have to think first whether they are useful for everybody...
So I have already forked OOM to my area; will I then just enter anything else that comes to my mind there and at some point make a pull request?

**Reply by dg0yt** [reply]
> So I have already forked OOM to my area; will I then just enter anything else that comes to my mind there and at some point make a pull request?

Yes. We can easily comment even single lines in this way. Individual users might still download the raw file and test it independently.
